operator/pkg/labels/labels.go

```diff
@@ -30,6 +30,11 @@ const (
 	PartOfKey = "app.kubernetes.io/part-of"
 	// The tool being used to manage the operation of an application
 	ManagedByKey = "app.kubernetes.io/managed-by"
+	// NodePoolKey is used to document the node pool associated with the StatefulSet.
```

Comment:
```suggestion
	// NodePoolKey is used to denote the node pool associated with the StatefulSet.
```

Struggling to explain why document isn't the right word here. I think document implies a separation of information and thing that's being informed about. A label on something wouldn't document something about itself but rather denote something about itself.

Reply:
updated